docs: Highlight the need for Istio configuration to get access logs (#814)
kyma-project/telemetry-manager: docs/user/integration/sap-cloud-logging/README.md

### Set Up Access Logs

> By default, Istio sidecar injection and Istio access logs are disabled in Kyma. With the following steps, you can analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.

Reviewer: ```suggestion
The Istio module is added to your Kyma cluster by default, but with Istio sidecar injection and Istio access logs disabled. 

To analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime, you must enable them:
```

Reviewer 2: I really think it is not important to talk about the istio module in general but only about if you enabled it for your workload. I somehow like the original better

Reviewer: If we make sure that Istio module is mentioned as a prerequisites for ingesting Istio logs etc, then no need to mention it here.

Reviewer: ```suggestion
By default, Istio sidecar injection and Istio access logs are disabled in Kyma.

To analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime, you must enable them:
```

### Set Up Access Logs

> By default, Istio access logs are disabled in Kyma. After following below procedure, you can analyze access logs in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.

Reviewer: ```suggestion
By default, Istio sidecar injection and Istio access logs are disabled in Kyma. With the following steps, you can analyze access logs of your workload in the default SAP Cloud Logging dashboards shipped for SAP BTP, Kyma runtime.
```

> 1. Enable Istio access logs for your workload following [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).

Reviewer 2: ```suggestion
1. Enable Istio sidecar injection for your workload following [Enabling Istio Sidecar Injection](https://kyma-project.io/#/istio/user/operation-guides/02-20-enable-sidecar-injection)

1. Enable Istio access logs for your workload following [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
```
